content/mdnotes.js

```diff
@@ -92,22 +92,15 @@ const typemap = {
   webpage: "Webpage",
 };
 
-function day_of_the_month(d) {
-  return (d.getDate() < 10 ? "0" : "") + d.getDate();
-}
-
-function get_month_mm_format(d) {
-  return (d.getMonth() < 10 ? "0" : "") + d.getMonth();
+function getDate(date) {
+  const d = new Date(date);
+  return d.toISOString().substring(0,10);;
```

Comment:
I seem to remember having tried this option and having had issues due to timezones.

Reply:
Since the timezone used in `toISOString` is UTC, this can cause issues depending on the time of day and the timezone of the user. See [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString) for more info.